### PR TITLE
Set `trailingComma` default value to `es5` 

### PR DESCRIPTION
**Pull request by fisker**
#6910

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by fisker** [comment]
@evilebottnawi come on it's 2019 still some `engines.node` requires `none`

**Comment by evilebottnawi** [comment]
@fisker no, but some people can write es3 code in theory can inserting trailing commas can break code, but if we don't support es3 we can set `es5` by default, but for better 0CJS we can respect node version

**Comment by fisker** [comment]
Self checked, didn't find any wrong result.

This is hard to review, but I think it's ready

File: src/language-js/options.js [hunk 1]
```diff
@@ -76,16 +76,17 @@ module.exports = {
...
     ],
     description: "Print trailing commas wherever possible when multi-line.",
     choices: [
-      { value: "none", description: "No trailing commas." },
       {
         value: "es5",
         description:
           "Trailing commas where valid in ES5 (objects, arrays, etc.)"
       },
+      { value: "none", description: "No trailing commas." },
```
Comment (ExE-Boss) [inline src/language-js/options.js]: https://github.com/prettier/prettier/pull/6910#pullrequestreview-314631630

Comment (fisker) [reply src/language-js/options.js]: https://github.com/prettier/prettier/pull/6910#discussion_r346168674

> @ExE-Boss why? after changing default to es5, shouldn't we put it on the top?

File: tests/angular_component_examples/jsfmt.spec.js [hunk 2]
```diff
@@ -1,2 +1,2 @@
+run_spec(__dirname, ["typescript"], { trailingComma: "none" });
```
Comment (fisker) [inline tests/angular_component_examples/jsfmt.spec.js]: https://github.com/prettier/prettier/pull/6910#discussion_r344485493

> I change the test orders to help review, snap changes should only be `options` part

**Comment by lydell** [comment]
@evilebottnawi We already turned down #6936 so adding automatic support for ES3 here won’t make sense. I think we should keep things simple. `es5` is always the default.

**Comment by lipis** [comment]
So merginnnnn?

**Comment by evilebottnawi** [comment]
Let's do it

**Comment by fisker** [comment]
👍

**Comment by kachkaev** [comment]
@fisker I'm working on #6929 and am about to create `changelog_unreleased/api/pr-7430.md` as asked in https://github.com/prettier/prettier/pull/7417#issuecomment-577651439. By looking at your PR, I cannot find a corresponding entry to use it as an example. If I simply can't find it, could you please share a link? Otherwise, could you please add a corresponding blogpost section for your change? I'll write mine is a similar style.

**Comment by fisker** [comment]
@kachkaev I didn't wrote one, if it's required, hope someone can help me out. I'm really bad at docs and English :(

**Pull request merged**